cl/recap/tests.py

```diff
@@ -913,6 +917,17 @@ def test_nothing_already_exists(self) -> None:
         self.assertEqual(self.pq.status, PROCESSING_STATUS.QUEUED_FOR_RETRY)
         self.assertIn("Unable to find docket", self.pq.error_message)
 
+    def test_ocr_extraction_recap_document(self):
+        """Can we extract a recap document via OCR?"""
+        cf = ContentFile(self.file_content_ocr)
+        self.pq.filepath_local.save(self.filename_ocr, cf)
+        process_recap_pdf(self.pq.pk)
+        recap_document = RECAPDocument.objects.all()
```

Comment:
I forget exactly how tests and transactions work, but this feels fragile, right? Maybe make process_recap_pdf return the `rd`, or look up the item more carefully?

Reply:
Yeah, I've updated the test to look for the recap document more carefully.

I used a similar approach for `RecapEmailDocketAlerts` tests but seems it's safe since we're using [TestCase](https://docs.djangoproject.com/en/4.1/topics/testing/tools/#django.test.TestCase) so each test runs inside of a transaction, so they're isolated.

Reply:
OK, I couldn't remember how those transactions work. Thanks for reminding me.